File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task136_PERPLEXITY_llama-3-sonar-large-32k-chat.java
// Java code
import java.util.Scanner;

public class Authentication {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter username: ");
        String username = scanner.nextLine();
        System.out.print("Enter password: ");
        String password = scanner.nextLine();

        if (username.equals("admin") && password.equals("password")) {
            System.out.println("Login successful!");
        } else {
            System.out.println("Invalid username or password");
        }
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task136_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
// C++ code
#include <iostream>
#include <map>
#include <string>

int main() {
    std::map<std::string, std::string> users;
    users["admin"] = "password";
    users["user"] = "password";

    std::string username, password;
    std::cout << "Enter username: ";
    std::cin >> username;
    std::cout << "Enter password: ";
    std::cin >> password;

    if (users.find(username) != users.end() && users[username] == password) {
        std::cout << "Authentication successful!" << std::endl;
    } else {
        std::cout << "Invalid username or password" << std::endl;
    }
    return 0;
}